<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralOptionsPage</class>
 <widget class="QWidget" name="GeneralOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="miscTab">
      <attribute name="title">
       <string>Miscellaneous</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="showZoomSliderButton">
            <property name="text">
             <string>Show zoom slider</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showMenuBarButton">
            <property name="text">
             <string>Show menu bar</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="compressLevelLabel">
              <property name="toolTip">
               <string>This must be in the range [0,100] or -1. Specify 0 to obtain small compressed files,
100 for large uncompressed files, and -1 to use the default settings.</string>
              </property>
              <property name="text">
               <string>Compression level for saved images:</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::NoTextInteraction</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="compressLevelSPB">
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>-1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="RFLabel">
              <property name="text">
               <string>Recent files showed:</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::NoTextInteraction</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="recentFilesNumberSPB">
              <property name="maximum">
               <number>15</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="deletRecentFiles">
              <property name="toolTip">
               <string>Once you close this window, the list can't be restored</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::RichText</enum>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="openExternalLinks">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="zoomincre">
              <property name="text">
               <string>Zoom increment value:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="zoomIncrementSPB">
              <property name="suffix">
               <string> %</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="toolTip">
               <string>Sets the precision of the file size number in the file properties dialog</string>
              </property>
              <property name="text">
               <string>File size precision:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="precisionNumberSpinBox">
              <property name="value">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Order files by:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="orderByComboBox">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="configureEggsLabel">
            <property name="text">
             <string notr="true">egg</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="bgTab">
      <property name="toolTip">
       <string/>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="accessibleName">
       <string>Images Background</string>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <attribute name="title">
       <string>Alpha channel background</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="bgToShowLabel">
             <property name="text">
              <string>Type:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="BGTypeComboBox">
             <item>
              <property name="text">
               <string>Checkered board</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Solid color</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="colorLabel">
             <property name="text">
              <string>Color:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="colorButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="squaresSizeLabel">
             <property name="text">
              <string>Size of the sides:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QSpinBox" name="squaresSizeSPB">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="suffix">
                <string> pixels</string>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>32</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="squaresSampleLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>34</width>
                 <height>34</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>34</width>
                 <height>34</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>34</width>
                 <height>34</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gifTab">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <attribute name="title">
       <string>Animated Images</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="stopWhenFinishButton">
         <property name="text">
          <string>Stop image reproduction when finish</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="restartWhenZoomingButton">
         <property name="text">
          <string>Restart animation when zooming</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="movieSpeedLabel">
           <property name="text">
            <string>Movie's speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="movieSpeedSPB">
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>50</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
